src/components/App.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+
+import Landing from './main/Landing';
+import Login from './auth/Login';
+import TodosList from './todos/TodosList';
+import TodoCreate from './todos/TodoCreate';
+import TodoEdit from './todos/TodoEdit';
+import TodoDelete from './todos/TodoDelete';
+import MainBar from './main/MainBar';
+import UserInfo from './main/UserInfo';
+
+
+function App() {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<Landing />} />
+        <Route path="/auth" element={<Login />} />
```

Comment:
See react docs for best-practice pattern for react router:
https://v5.reactrouter.com/web/guides/quick-start

Reply:
Your link refers to react-router v5, the pattern is different in v6:
[https://reactrouter.com/docs/en/v6/upgrading/v5](url)